### PR TITLE
[ETL-275] Set up JumpCloud access for RECOVER Prod account

### PR DESCRIPTION
**Pull request by rxu17**
Hi,
This PR builds on top of now merged [#650](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/650) and sets up access to the AWS account via JumpCloud.
Ideally, admin, developer and viewer roles are created.

A blank placeholder document (for the purpose of creating the initial PR and will be deleted later prior to merging) has been generated at _/.../org-formation/600-access/etl_275_blank_file.txt

PR Checklist:
[X] Describe and explain your intentions for this change
[X] Setup pre-commit and run the validators (info in README.md)

### CODE REVIEW

**Comment by thomasyu888** [comment]
@brucehoff , @zaro0508 The purpose of this is to add in the XML document that is required for connection to jumpcloud

**Comment by zaro0508** [comment]
i'm not sure what this PR is about. If you want to setup access via jumpcloud then you'll need to add something similar to this.. https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L1245-L1294

**Comment by rxu17** [comment]
Updated

**Comment by thomasyu888** [comment]
@rxu17 can you remove the empty file?

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -1309,3 +1309,54 @@ Sso8dxfh53Admin:
...
+  StackDescription: 'SSO: Administrator role used by recover admin group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref RecoverProdAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref recoverProdAdminGroup
```
Comment (zaro0508) [inline org-formation/700-aws-sso/_tasks.yaml:1327]: please add this to the `Parameters` section above 

```
  recoverProdAdminGroup:      #JC aws-recover-prod-admins
    Type: String
    Default: '54d83448-10a1-7000-7e72-cae28f0caacf'

  recoverProdDeveloperGroup:      #JC aws-recover-prod-developers
    Type: String
    Default: 'a468b478-f021-7010-535b-4ad779417559'

  recoverProdViewerGroup:      #JC aws-recover-prod-viewers
    Type: String
    Default: 'b4e86468-f091-70e2-0ea9-64cf3addcdf1'
```

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (rxu17) [reply org-formation/700-aws-sso/_tasks.yaml:1327]: Done!

**Comment by brucehoff** [comment]
> Got a request to reset the password to this and the dev AWS account. Should I ignore it?

Yes, please ignore it, @rxu17 .

**Review by brucehoff — APPROVED** [review]

**Comment by brucehoff** [comment]
@zaro0508 It looks like this PR is needed for steps 9 and 10 of [these](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/745373697/Setup+AWS+Accounts) instructions.  The PR looks good to me, but would you please also review before it is merged?

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/700-aws-sso/_tasks.yaml:1327]: sorry, i didn't mean add here.  The `Parameters:` section is from line 1 to  about 200.  Please add these parameters with somewhere in that section with the other JC group parameters.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (rxu17) [reply org-formation/700-aws-sso/_tasks.yaml:1339]: Done. Does this look correct now?

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**